packages/crw-wallet/Cargo.toml

```
@@ -26,7 +26,7 @@ parking_lot = { version = "0.11.1", default-features = false, features = ["wasm-
 rand = { version = "0.7.3", features = ["wasm-bindgen"], optional = true }
 libc = { version = "0.2.94", optional = true }
 ffi_helpers = { version = "0.3.0", optional = true }
-
+getrandom = { version = "0.2", features = ["js"] }
```

```suggestion
getrandom = { version = "0.2.7" }

```
Let's use the explicit syntax for the depency like the other ones and please remove the empty line above.
Also since the `js` feature of `getrandom` is used only when compiling for web-assembly you can add `getrandom/js` inside the array of `wasm-bindigen` at line 43 to enable it only when compiling the wasm package